assigner/baserepo.py

```diff
@@ -332,27 +331,30 @@ def archive(self):
     def unarchive(self):
         return self._gl_post("/projects/{}/unarchive".format(self.id))
 
-    def protect(self, branch="master", developer_push=True, developer_merge=True): # NOTE: these are not the same defaults that Gitlab uses
+    # NOTE: these are not the same defaults that Gitlab uses
+    def protect(self, branch="master", developer_push=True, developer_merge=True):
         params = {
             "developers_can_push": developer_push,
             "developers_can_merge": developer_merge,
-          }
-        return self._gl_put("/projects/{}/repository/branches/{}/protect".format(self.id, branch), params)
+        }
+        return self._gl_put("/projects/{}/repository/branches/{}/protect"
+                            .format(self.id, branch), params)
 
     def unprotect(self, branch="master"):
-        return self._gl_put("/projects/{}/repository/branches/{}/unprotect".format(self.id, branch))
+        return self._gl_put("/projects/{}/repository/branches/{}/unprotect"
+                            .format(self.id, branch))
 
     def _gl_get(self, path, params={}):
         return self.__class__._cls_gl_get(
             self.url_base, path, self.token, params
         )
 
-    def _gl_post(self, path, payload={}, params={}):
+    def _gl_post(self, path, payload={}, _=None):
```

Comment:
We should actually probably be passing `params` through to `_cls_gl_post` here...

Reply:
Yep, done. This is when happens when you fix lint without reading code.